circle.yml

```diff
@@ -7,6 +7,15 @@ test:
     - gulp -v
   override:
     - npm test # Run the package and docs test suite
-    - npm run build:package # Run the release process
+    - npm run build
   post:
-    - ls -agolf dist/ # Ensure that build:package worked
\ No newline at end of file
+    - ls -agolf dist/ # Ensure that build:package worked
+
+deployment:
+  npm:
+    branch: master
```

Comment:
Do we use git tags for this at all? So we publish a git tag (which shows up on the github releases page) and then publish to npm?